tests/artist-read.test.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef */
+const { expect } = require("chai");
+const request = require("supertest");
+const db = require("../db/index");
+const app = require("../src/app");
+
+describe("Read Artists", () => {
+  let artists;
+
+  beforeEach(async () => {
+    const responses = await Promise.all([
+      db.query(
+        "INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *",
+        ["Tame Impala", "rock"]
+      ),
+      db.query(
+        "INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *",
+        ["Kylie Minogue", "pop"]
+      ),
+      db.query(
+        "INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *",
+        ["Tame Antelope", "jazz"]
+      ),
+    ]);
+
+    artists = responses.map(({ rows }) => rows[0]);
+  });
+
+  describe("GET /artists", () => {
+    xit("returns all artist records in the database", async () => {
```

Comment:
This test is failing, I see why you skipped it. It could be because the test environment isnt being cleared properly, I would have a word with one of the back end tutors specifically about this as the artists you created as part of the album-create-test are still being included here